projects/ngneat/reactive-forms/src/lib/formGroup.ts

```diff
@@ -348,7 +348,7 @@ export class FormGroup<T extends Obj = any, E extends object = any> extends NgFo
       tap(value => {
         if (!value) return;
         handleFormArrays(this, value, arrControlFactory);
-        this.patchValue(value, { emitEvent: false });
+        this.patchValue(value as any, { emitEvent: false });
```

Comment:
do we have any reference of why this is failing now?

Reply:
@rafaelss95

Reply:
Hmm, I didn't go deep but I faced this typing error:

![Captura de Tela 2020-11-26 às 11 17 12](https://user-images.githubusercontent.com/11965907/100361629-262faf00-2fd9-11eb-94d5-6c1732ebfa8e.png)

Let me know if you have a better solution.

Reply:
sure, let's try to find out why might be typescript's bug if so let's reference the issue

Reply:
I don't think it's a Typescript bug, it happens because `value` in this context is just typed as `Obj` (`{ [key: string]: any }`), however `patchValue` expects an `Observable<DeepPartial<ControlsValue>>` or an `DeepPartial<ControlsValue>` and `Obj` isn't assignable to `ControlsValue`.

Reply:
your explanation doesn't make sense at all, if that's true then how come it worked before?

Reply:
Well, I didn't change `patchValue` signature directly, but indirectly. Anyway, as I had some more free time today I was able to fix it by changing the conditional in `DeepPartial`. Let me know if we can finally move on with this. Regards.